packages/react-native-payments/lib/js/NativePayments.js

```diff
@@ -77,7 +77,7 @@ const NativePayments: {
     });
   },
 
-  handleDetailsUpdate(details) {
+  handleDetailsUpdate(details, invalidShippingAddress) {
```

Comment:
also @andiskim does the type have to be updated above where it says `handleDetailsUpdate: PaymentDetailsBase => Promise<any>`. So does `PaymentDetailsBase` have to be updated?

`

Reply:
hmmm I think there needs to be separate boolean type because PaymentDetailsBase the type for the first parameter

Reply:
Right at this point it's just an invalid type signature